examples/medplum-provider/src/components/messages/ChatListItem.tsx

```diff
@@ -12,11 +12,11 @@ interface ChatListItemProps {
   topic: Communication;
   lastCommunication: Communication | undefined;
   isSelected: boolean;
-  onSelectedItem: (topic: Communication) => string;
+  getThreadUri: (topic: Communication) => string;
```

Comment:
Refactoring this name looks like it was more in-depth than I would have guessed, but I think it makes the intention much clearer. Thank you for all this work!